Component-assets/Css/ComponentsStyles.css

```diff
@@ -593,109 +590,190 @@ cursor: pointer;
-color: blueviolet;
+.card__button--secondary {
+  margin-left: 0.2rem;
+  background-color: transparent;
+  color: blueviolet;
 }
-.card__button--primary:hover{
+.card__button--primary:hover {
   background-color: transparent;
   color: blueviolet;
 }
-.card__button--secondary:hover{
+.card__button--secondary:hover {
   background-color: blueviolet;
   color: white;
 }
-.card__badge{
-  background-color:rgb(255, 255, 255);
+.card__badge {
+  background-color: rgb(255, 255, 255);
   display: inline-block;
   padding: 0.2rem;
   font-weight: 900;
   color: rgb(0, 0, 0);
   border-radius: 0.2rem;
   position: absolute;
-  top:1rem;
+  top: 1rem;
   left: 1rem;
 }
 /*card Dismiss */
 
-.card-dismiss--title{
+.card-dismiss--title {
   background-color: blueviolet;
   padding: 0.3rem;
   border-radius: 0.5rem;
   color: white;
 }
-.card-dismiss--text{
+.card-dismiss--text {
   border: 2px solid blueviolet;
   padding: 0.5rem;
   border-radius: 0.5rem;
 }
-.card-dismiss--icon{
-  background-color:transparent;
-  border:none;
+.card-dismiss--icon {
+  background-color: transparent;
+  border: none;
   position: absolute;
-  top:0.8rem;
+  top: 0.8rem;
   right: 0.8rem;
 }
-.card-dismiss__img{
-  width:1.5rem;
-  height:1.5rem;
+.card-dismiss__img {
+  width: 1.5rem;
+  height: 1.5rem;
 }
-.card-dismiss--icon:hover{
+.card-dismiss--icon:hover {
   background-color: rgba(252, 252, 252, 0.308);
-  border-radius:0.2rem;
+  border-radius: 0.2rem;
   display: flex;
 }
 /* Text Overlay */
-.card__content--overlay{
+.card__content--overlay {
   position: absolute;
-  top:10.8rem;
-  left:1rem;
-  color:white;
+  top: 10.8rem;
+  left: 1rem;
+  color: white;
 }
 /* Card horizontal */
 
-.card-hr__wrapper{
+.card-hr__wrapper {
   display: flex;
   flex-direction: row;
   width: 100%;
   max-width: 25rem;
   min-width: 12rem;
 }
-.card-hr__img--wrapper{
+.card-hr__img--wrapper {
   width: 50%;
 }
-.card-hr__content{
+.card-hr__content {
   margin: 0 8px;
   width: 50%;
 }
-.card-hr__img{
+.card-hr__img {
   width: 100%;
   flex-shrink: 1;
-}
\ No newline at end of file
+}
+/*Text Utilities*/
+.line-start {
+  text-align: start;
+}
+.line-center {
+  text-align: center;
+}
+.line-end {
+  text-align: end;
+}
+.gray-text {
+  color: gray;
+}
+
+/*List*/
+.spaced-list {
+  border: 2px solid skyblue;
+  width: fit-content;
+  text-align: center;
+  list-style-type: none;
+  padding: 0.5rem 1rem;
+}
+.spaced-list--items {
+  margin: 0.5rem;
+}
+
+.linked-list {
+  border: 2px solid var(--primary-bg);
+  width: fit-content;
+  text-align: center;
+  list-style-type: none;
+  padding: 0.5rem 1rem;
+}
+.linked-list--items {
+  margin: 0.5rem;
+}
+.stacked-list {
+  border: 2px solid var(--primary-bg);
+  width: fit-content;
+  text-align: center;
+  list-style-type: none;
+  padding: 0.5rem 1rem;
+}
+.stacked-list--items {
+  border: 2px solid var(--secondary-bg);
+  margin: 4px;
+  padding: 4px;
+}
+
+/* Rating */
+.rating {
+  display: flex;
```

Comment:
I found this display flex property to be very repetitive in your code. You could make a utility class out of it.

Reply:
okay, I will try to make changes.